.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+- 3.6
+before_install:
+- touch .env
+install:
+- python setup.py install
+script:
+- start
```

Comment:
These are the steps taken from the README, but apparently it doesn't work yet. Are there more steps to take in order to run the bot?

Reply:
Thats shouldn't be the case. According to the Stacktrace it might be a problem with line-endings